src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(java.lang.String[] args) {
+        Figure[] figures = new Figure[6];
+        FigureSupplier figureSupplier = new FigureSupplier();
+        for (int i = 0; i < figures.length / 2; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+        }
+        for (int y = figures.length / 2; y < figures.length; y++) {
```

Comment:
Optional: it is common to use 'i' then 'j' as a variable name in for cycle, but you can you 'i' here as well

Reply:
Also to make code more readable it makes sense to create a method something like 'polulateFigures' or 'getFigures' and put the logic of creating and populating array there